schematools/contrib/django/factories.py

```diff
@@ -93,6 +93,26 @@ def handle_relation(
 
             if field._parent_table.has_parent_table:
                 kwargs["related_name"] = field._parent_table["originalID"]
+            else:
+                related_name = None
+                _, related_table_name = relation.split(":")
+                relations = {}
+                for table in dataset.tables:
+                    if table.id == related_table_name and table.relations:
+                        relations = table.relations
+                        break
+                if relations:
+                    for name, relation in relations.items():
+                        if (
+                            relation["table"] == field.table.id
+                            and relation["field"] == field.name
+                        ):
+                            related_name = name
+                            break
+                if related_name:
+                    kwargs["related_name"] = name
+                else:
+                    kwargs["related_name"] = "+"
```

Comment:
```suggestion
                    kwargs["related_name"] = f"+{name}"
```

Als er meerdere zijn moet de name toch uniek zijn, ondanks dat deze verborgen wordt.

Reply:
Die name is dan niet beschikbaar, en ik zie het ook niet de documentatie date related_name uniek moet zijn. Als related_name  niet gezet is is de default <model_name>_set ook niet uniek voor meerdere forward relaties naar dezelfde tabel